TGM/src/main/java/network/warzone/tgm/modules/ChatModule.java

```diff
@@ -53,14 +53,21 @@ public void onCommand(PlayerCommandPreprocessEvent event) {
     }
 
     private boolean startsWith(String msg) {
-        for (String cmd : blockdCmds) {
+        for (String cmd : blockedCmds) {
             if (msg.toLowerCase().startsWith("/" + cmd) || msg.toLowerCase().startsWith(cmd)) return true;
         }
         return false;
     }
 
     @EventHandler(priority = EventPriority.LOW)
     public void onChat(AsyncPlayerChatEvent event) {
+        // Run this code if the chat is currently muted
+        if (!TGM.get().getConfig().getBoolean("chat.enabled") && !event.getPlayer().hasPermission("tgm.chat.bypass")) {
+            event.getPlayer().sendMessage(ChatColor.RED + "Chat is currently disabled. If you believe this is an error, please contact a staff member on our Discord.");
```

Comment:
Can we add this message to the config? We're eventually going to have translations but I feel like we should have this configurable